.kokoro/samples/python3.6/periodic-head.cfg

```diff
@@ -0,0 +1,11 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+env_vars: {
+    key: "INSTALL_LIBRARY_FROM_SOURCE"
+    value: "True"
+}
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/python-pubsub/.kokoro/test-samples-against-head.sh"
```

Comment:
```suggestion
    value: "github/python-aiplatform/.kokoro/test-samples-against-head.sh"
```

Reply:
Great catch! It's strange to see pubsub here. I was working with pubsub yesterday but I'm not sure how the changes landed here. [This is the commit](https://github.com/googleapis/python-aiplatform/commit/a0cbbe09ba036591ff4c24c2c623b05928667b84) where the changes were added. I'm looking into how it happened.

Reply:
It looks like python-pubsub is hardcoded in synthtool templates here: https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/.kokoro/samples/python3.6/periodic-head.cfg#L10

Reply:
Will be fixed in https://github.com/googleapis/synthtool/pull/1167

Reply:
Fixed in https://github.com/googleapis/python-aiplatform/pull/175/commits/437b5a138ef135353b08d12ae1a217924d4cacfe